src/components/radio-button/radio-button.jsx

```diff
@@ -158,6 +158,7 @@ const RadioButtonComponent = (
 		hideSelection = false,
 		reversePosition = false,
 		borderOn = false,
+		borderOnActive = true,
```

Comment:
Let's not add one more prop here.
We can get rid of it altogether.

If there is any such requirement we will add it.
You can simply remove the related classes.